dione-hadoop/src/main/scala/com/paypal/dione/hdfs/index/avro/AvroIndexer.scala

```diff
@@ -63,21 +63,15 @@ case class AvroIndexer(file: Path, start: Long, end: Long, conf: Configuration)
   }
 
   override def getCurMetadata(): HdfsIndexerMetadata = {
-    val size = tryInferSize(fileReader, lastPosition, totalInBlock.intValue())
+    val size = tryInferSize(fileReader, totalInBlock.intValue())
     lastPosition = fileReader.previousSync()
     HdfsIndexerMetadata(file.toString, prevSync, numInBlock, size)
   }
 
   def getSchema(): Schema = fileReader.getSchema
 
-  private def tryInferSize(reader: DataFileReader[GenericRecord], lastPosition: Long, totalInBlock: Int): Int = {
-    val pos = reader.previousSync()
-    val longSize =
-      if (totalInBlock == 1) // exactly 1 row in block
```

Comment:
@shay1bz - do you happen to remember why we had this in the first place?

Reply:
I think I remember that dividing block size with block count (as in PR) failed on some edge case - but I really don't remember which =\ maybe first/last block or something like that